### PR TITLE
Added installation instructions

### PR DESCRIPTION
**Pull request by xoltar**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [flake8]
+max-line-length: 120
```
Comment (mihaic) [inline setup.cfg]: I think we should have uniform settings in brainiak and brainiak-extras. If you want to merge the code now and reformat it later, please exclude the files that fail the checks and make an issue to reformat them later.

Comment (xoltar) [inline setup.cfg]: Makes sense. 80 seems absurdly small for current monitors. I'll submit a 120 PR for brainiak.

Comment (mihaic) [inline setup.cfg]: We had this discussion when we started the project. The number is not based on monitor size, but on human comprehension ability.

Comment (xoltar) [inline setup.cfg]: Hmm. I don't recall that discussion, and have never been in favor of 80 column limits, so it seems unlikely I was there. However I don't wish to re-legislate if it's already been decided. I'll change this one.

**Pull request merged**